apps/web/components/email-list/EmailList.tsx

```diff
@@ -472,6 +488,28 @@ export function EmailList(props: {
                   />
                 );
               })}
+              {/* Load more button to fetch more data */}
+              {showLoadMore && (
+                <div className="pb-2">
+                  <Button
+                    variant="outline"
+                    className="w-full py-4"
```

Comment:
is py-4 needed? there's button size prop if you want to adjust sizing but not sure that's needed here

Reply:
Added size prop and removed py-4